orc8r/cloud/go/services/orchestrator/obsidian/handlers/network_handlers.go

```diff
@@ -53,6 +53,22 @@ func registerNetwork(c echo.Context) error {
 	if err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, err)
 	}
+
+	// Creating default tier for each created network
+	for _, s := range createdNetworks {
```

Comment:
`s` is a little under-descriptive for me. I'd prob just go with `network` (normally I'd say `n`, but `n` can be confused with a number)